examples/imgui_impl_dx9.cpp

```diff
@@ -368,6 +374,8 @@ static void ImGui_ImplDX9_RenderWindow(ImGuiViewport* viewport, void*)
 static void ImGui_ImplDX9_SwapBuffers(ImGuiViewport* viewport, void*)
 {
     ImGuiViewportDataDx9* data = (ImGuiViewportDataDx9*)viewport->RendererUserData;
+    if (data->SwapChain == NULL)
+        return;
     data->SwapChain->Present(NULL, NULL, data->d3dpp.hDeviceWindow, NULL, NULL);
```

Comment:
Looking at this PR now. What are those two NULL checks for?

Reply:
Oh sorry. I test it in my branch. This check is no need to be here now. Because I made a mistake some days ago. And I fixed it and forgot delete the check.
Thanks for review, I will commit modify right now.